application/tests/features/api.messages.feature

```diff
@@ -224,3 +225,20 @@ Feature: Testing the Messages API
         Then the response is JSON
         And the "id" property equals "110"
         Then the guzzle status code should be 200
+
+    Scenario: Admin can set user id when creating a message
```

Comment:
TBH I'm not 100% this should be the case. Its what we did for posts. But it might actually be simpler if CreateMessage just **always** set user id to the current user..